_posts/2024-12-10-development-and-debugging-of-docker-services-(part-1)--open-mic-summary.markdown

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
for filename - my best practice is to use the date of openmic session, for retrospective (ofc not fopr announcements)

Reply:
ok i will fix it in the script for next time